packages/core/src/common/logger.ts

```diff
@@ -18,6 +18,20 @@ export enum LogLevel {
     TRACE = 10
 }
 
+export const logLevelStrings = new Map([
```

Comment:
please put exported global variables and functions, under namespace of the related main concept, i.e. `LogLevel` namespace:
```ts
export namespace LogLevel {
    export const strings = new Map([...]);
    export function toString(level: LogLevel): string | undefined { ... }
}
```

Reply:
Done.